README.md

```diff
@@ -38,17 +38,18 @@ The following sample creates an authentication provider that [gets access withou
 ```php
 
 use Microsoft\Kiota\Authentication\Oauth\ClientCredentialContext;
-use Microsoft\Graph\Core\Authentication\GraphPhpLeagueAuthenticationProvider;
+use Microsoft\Kiota\Authentication\PhpLeagueAuthenticationProvider;
 
 $tokenRequestContext = new ClientCredentialContext(
     'tenantId',
     'clientId',
     'clientSecret'
 );
 // uses https://graph.microsoft.com/.default scopes
-$authProvider = new GraphPhpLeagueAuthenticationProvider($tokenRequestContext);
+$authProvider = new PhpLeagueAuthenticationProvider($tokenRequestContext);
 
 ```
+> **Note:** Don't forget to add the PHP opening tag (<?php) at the beginning of your file.
```

Comment:
We could add opening tags at the beginning of every code sample here. Thoughts?

Reply:
I like this idea better, thank you